distribution/docs/src/main/resources/content/_configuring/from-import-intro.adoc

```diff
@@ -9,15 +9,18 @@ The Configuration Export/Import capability allows administrators to export the c
 To export the current configuration settings:
 
 . Run the command migration:export from the ${command-console}.
-. A file named `exported-${project.version}.zip` will be created in the `exported` directory underneath `<${branding}_HOME>`, but an alternative location can be specified when invoking the command. Copy this file to a secure place.
+. A file named `exported-${project.version}.zip` will be created in the `exported` directory underneath `<${branding}_HOME>`. Copy this file to a secure place.
 
 To import previously exported configuration settings:
 
+. Install ${branding}
+. Restore all external files, softlinks, and directories that would not have been exported and for which warnings would have been generated during export. This could include among others, things like external certificates, content store, monitored directories.
 . Launch the newly installed ${branding}.
 . Step through the installation process.
 . Make sure to re-enable <<_automatic_start_on_system_boot,automatic startup and shutdown>> of ${branding} on the new system if it was enabled on the originating system.
-. Copy the previously exported file from your secure place to the `exported` directory underneath `<${branding}_HOME>`, but an alternative location can be specified when invoking the command. Create the directory if it does not exist.
+. Copy the previously exported file from your secure place to the `exported` directory underneath `<${branding}_HOME>`. Create the directory if it does not exist.
 . Run the command migration:import from the ${command-console}.
+. ${branding} will automatically restart if the command is successful. Otherwise address any generated warnings before manually restarting ${branding}.
```

Comment:
:question: After the solr 6.6.1 upgrade, we force a restart after installation.  Will a forced restart happen after import?

Reply:
correct. Once we have the profile migratable in place, there will be only one restart.